skills/lark-drive/references/scripts/publish_gate.py

```
@@ -0,0 +1,338 @@
+    """Evaluate a source_attachment item.
+
+    An attachment is the original file uploaded via drive +upload. It never
+    carries a governance table and never counts as a completed knowledge page;
+    it requires an explicit user opt-in (attachment_confirmed).
+    """
+    hard_reasons: list[str] = []
+    write_via = str(item.get("write_via") or "")
+
+    if write_via != ATTACHMENT_WRITE_VIA:
+        hard_reasons.append(
+            f"source_attachment 只能用 drive_upload，当前 write_via={write_via or '（空）'}"
```

Ruff (0.16.3)

[warning] String contains ambiguous `，` (FULLWIDTH COMMA). Did you mean `,` (COMMA)?

(RUF001)